addon/orm/model.js

```diff
@@ -221,6 +223,7 @@ class Model {
 
     // Define the getter/setter
     Object.defineProperty(this, attr, {
+      configurable: true,
```

Comment:
To make the test I added pass, it was necessary to remove the checks before Object.defineProperty, which means it will be run every time. Should be fine to do that I suppose.